go.mod

```diff
@@ -368,6 +367,7 @@ require (
 	google.golang.org/appengine v1.6.7 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
why is this changing?

Reply:
oh I know, b/c RDK likely no longer directly uses yaml, after deleting the orbslam_yaml.go file.

Reply:
This was automatically updated when running `go mod tidy`. My suspicion is that because we're deleting orbslam_yaml.go, we stop importing it directly, but it is probably still required as a dependency in another package we're importing.